java/ql/lib/semmle/code/java/security/IntentUriPermissionManipulation.qll

```diff
@@ -123,3 +132,12 @@ private predicate bitwiseCheck(Guard g, boolean branch) {
     else g.(EqualityTest).polarity().booleanNot() = branch
   )
 }
+
+/**
+ * Holds if taint can flow from `source` to `sink` in one local step,
+ * including bitwise operations.
+ */
+private predicate bitwiseLocalTaintStep(Expr source, Expr sink) {
+  TaintTracking::localTaintStep(DataFlow::exprNode(source), DataFlow::exprNode(sink)) or
+  source = sink.(BinaryExpr).getAnOperand()
```

Comment:
You probably meant `sink.(BitWiseExpr).(BinaryExpr).getAnOperand()` instead of including all `BinaryExpr`s.

Reply:
You're absolutely right. Fixed in 3957ebe.